### PR TITLE
hashrat: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Hashing tool supporting md5, sha1, sha256, sha512, whirlpool, jh and hmac versions of these. Include recursive file hashing and other features.

### CODE REVIEW

**Comment by noraj** [comment]
It seems similar to rhash, which is in official AL repo, while being less powerful. I don't see the interest of packaging it.

**Comment by D3vil0p3r** [comment]
> It seems similar to rhash, which is in official AL repo, while being less powerful. I don't see the interest of packaging it.

As you prefer @noraj. I created this PR because I see this tool in the new Kali Purple release and I tried to bring it to BlackArch environment.

File: packages/hashrat/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+pkgname=hashrat
+pkgver=v1.15.r0.g0b3c057
+pkgrel=1
+pkgdesc='Hashing tool supporting md5, sha1, sha256, sha512, whirlpool, jh and hmac versions of these. Include recursive file hashing and other features.'
+arch=('any')
+groups=('blackarch' 'blackarch-cracker' 'blackarch-crypto')
+url='https://github.com/ColumPaget/Hashrat'
+license=('GPL')
+depends=()
+makedepends=('autoconf' 'automake' 'git')
```
Comment (noraj) [inline packages/hashrat/PKGBUILD:13]:
```suggestion
makedepends=('git')
```

cf. #2619

File: packages/hashrat/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hashrat
+pkgver=v1.15.r0.g0b3c057
+pkgrel=1
+pkgdesc='Hashing tool supporting md5, sha1, sha256, sha512, whirlpool, jh and hmac versions of these. Include recursive file hashing and other features.'
+arch=('any')
+groups=('blackarch' 'blackarch-cracker' 'blackarch-crypto')
```
Comment (noraj) [inline packages/hashrat/PKGBUILD:9]:
```suggestion
groups=('blackarch' 'blackarch-crypto' 'blackarch-forensic')
```

**Review by noraj — CHANGES_REQUESTED** [review]
Outside of those small changes, LGTM. Tested build, install, exec.

**Review by noraj — APPROVED** [review]

**Pull request merged**